src/main/java/io/github/aquerr/rentacar/web/rest/VehicleRestController.java

```diff
@@ -49,6 +50,7 @@ public ResponseEntity<VehicleAvailabilityResponse> isVehicleAvailable(@RequestPa
     }
 
     @PostMapping
+    @PreAuthorize("hasAuthority('VIEW_ADMIN_PANEL')")
```

Comment:
W zasadzie to może jednak `ADD_VEHICLE`?

Reply:
Ja bym splitował. Z doświadczenia wiem że potem w razie czego jesteś w dupie. 😆

Reply:
to ustawie add_vehicle i dorzuce do dummy data admina

Reply:
ok